pymmh3.py

```diff
@@ -14,18 +14,17 @@
 
 This module is written to have the same format as mmh3 python package found here for simple conversions:
 
-https://pypi.python.org/pypi/mmh3/2.0 
+https://pypi.python.org/pypi/mmh3/2.3.1
 '''
 
-import sys
-
-if (sys.version_info > (3, 0)):
+import sys as _sys
```

Comment:
That one is on me.. if we don't delete it, we can import sys from pymmh3.
I just could ignore it. My  i guess :P